Compiler/test2/cc_parser_test_0.cpp

```diff
@@ -56,7 +56,7 @@ int cc_compile(std::string const &inpl, AGS::ccCompiledScript &scrip)
     AGS::MessageHandler::Entry const &err = mh.GetError();
     static char buffer[256];
     ccCurScriptName = buffer;
-    strncpy_s(
+    strncpy(
```

Comment:
If strncpy_s cannot be used, it's better to use snprintf instead, because strncpy has unexpected quirks. For instance, it does not guarantee null-termination if string is longer than the buffer.
https://en.cppreference.com/w/c/string/byte/strncpy

Reply:
On another hand, the way ccCurScriptName is a horrible hack (and dangerous too), it must be investigated and possibly replaced with simply std::string. Maybe separately from this PR.

Reply:
strncpy_s is a Microsoft extension. We could make an equivalent somewhere and use that where it's used otherwise it won't work in non-MSVC compilers.

Reply:
ok, I wrote a better one using snprintf and abusing the format string, I think it's really forcing the buffer to be null-terminated and has no easy area for errors I think.